tests/cloudferrylib/os/actions/test_check_networks.py

```diff
@@ -14,16 +14,35 @@
 #    under the License.
 
 
+import os
+
 import mock
 
 from cloudferrylib.base import exception
+from cloudferrylib.utils import utils
 from cloudferrylib.os.actions import check_networks
 from tests import test
 
 
+RESULT_FILE = 'tests/check_net_result'
```

Comment:
Unit tests must not depend on external objects. This will be removed once you remove logic to store net check results.